Dockerfile

```diff
@@ -0,0 +1,27 @@
+# SET GO AND ALPINE VERSIONS
+ARG GO_VER=1.15.3
+ARG ALPINE_VER=3.12
+
+# COPY AND BUILD SOURCE
+FROM golang:${GO_VER}-alpine${ALPINE_VER} AS builder
+COPY . /src
+RUN cd /src/jira-ui && go build -o /tmp/jira main.go
+
+########## ########## ##########
+
+# START A LEAN CONTAINER
+FROM alpine:${ALPINE_VER}
+
+# COPY ARTIFACT FROM BUILDER CONTAINER
+COPY --from=builder /tmp/jira /usr/local/bin/jira
```

Comment:
Could you leave the tool name as 'jira-ui' please - it is a sister tool to the quite popular https://github.com/go-jira/jira project, so renaming creates a conflict on the tool name.